riomucho/__init__.py

```
@@ -8,88 +11,85 @@
 from riomucho.single_process_pool import MockTub
 import traceback
 
+
 work_func = None
 global_args = None
 srcs = None
 
 
 def main_worker(inpaths, g_work_func, g_args):
-    """"""
+    """TODO"""
     global work_func
     global global_args
     global srcs
     work_func = g_work_func
     global_args = g_args
-    try:
-        srcs = [rio.open(i) for i in inpaths]
-    except:
-        return
```

This swallows all exceptions and I think it's better to delete it.